controllers/toolchainstatus/toolchainstatus_controller_test.go

```diff
@@ -1724,6 +1724,26 @@ func TestGenerateUnreadyNotificationContent(t *testing.T) {
 	})
 }
 
+func TestRemoveSchemeFromURL(t *testing.T) {
+	t.Run("test proxy url domain extraction", func(t *testing.T) {
+		// when
+		domain, err := removeSchemeFromURL("https://api-toolchain-host-operator.apps.stone-stg-host.qc0p.p1.openshiftapps.com")
+
+		// then
+		require.Equal(t, "api-toolchain-host-operator.apps.stone-stg-host.qc0p.p1.openshiftapps.com", domain)
+		require.Nil(t, err)
+
+		// when
+		domain, err = removeSchemeFromURL("incorrect$%url")
+
+		// then
+		require.Equal(t, "", domain)
+		require.NotNil(t, err)
+
+	})
```

Comment:
thanks for adding the unit test! 

Could you please split it into 2 subtests, maybe something like "valid URL" and "incorrect URL?